core/src/main/java/com/klaytn/caver/wallet/keyring/MessageSigned.java

```diff
@@ -12,9 +14,9 @@
     String messageHash;
 
     /**
-     * ECDSA signature data(V, R, S)
+     * ECDSA signature data(V, R, S) list
      */
-    KlaySignatureData signatureData;
+    List<SignatureData> signatureData;
```

Comment:
```suggestion
    List<SignatureData> signatures;
```
Can you rename this?